### ISSUE TITLE
Chrome and Chromium won't use specified profile

### ISSUE THREAD

**Issue by nenadalm** [igrishaev/etaoin#120]
Hi. When I've tried to create profile as described in the readme, browsers started but they didn't load the profile (I enabled notifications and camera/microphone in the profiles).

```
$ chromedriver --version
ChromeDriver 2.35.528139 (47ead77cb35ad2a9a83248b292151462a66cd881)
$ chromium-browser --version
Chromium 63.0.3239.108 Fedora Project
$ google-chrome --version
Google Chrome 64.0.3282.140
```

**Comment by igrishaev** [comment]
Well, I don't know whether those settings are passed through profiles. I tested profiles by setting a custom theme. It might be an issue related with different folder structure of Linux.

**Comment by igrishaev** [comment]
@Uunnamed could you take a look at it, please? Maybe it's been fixed in the modern releases of the drivers. You need to create a custom profile for the browser with some specific settings (e.g. install extensions, change colour scheme) and then pass the path to the profile when running the code.

**Comment by Uunnamed** [comment]
@nenadalm thank you for your feedback. Fixed. 
The fix will be available since 0.3.9.
